docs(manifest): Improvements to app store members (#36605)
mdn/content: files/en-us/web/manifest/screenshots/index.md

title: screenshots
slug: Web/Manifest/screenshots
page-type: web-manifest-member
spec-urls: https://w3c.github.io/manifest-app-info/#screenshots-member
---

{{QuickLinksWithSubpages("/en-US/docs/Web/Manifest")}}

The `screenshots` manifest member lets you specify one or more images that showcase your web application.
These images help users preview your web app's interface and features in distribution platforms, such as Google Play Store.

> [!NOTE]
> The `screenshots` member is optional and purely advisory.

> > Distribution platforms may choose to ignore it when presenting your app.

Reviewer: You have used the text "member is optional and purely advisory." in a few places. I don't love the words " purely advisory", because if you're trying to say apps stores don't need to use it then you said that in the next line.

How about everywhere we do. 

``suggestion
> [!NOTE]
> The `screenshots` member is optional, and the images may not be displayed by application distribution platforms.
```

Author: I guess I picked the verbiage from the spec but I agree it can be much simpler.

Going with:
- "Note: The screenshots member is optional, and app stores may not display these images when presenting your app."
- "Note: The description member is optional, and app stores may not use this text when presenting your app."
- "Note: The categories member is optional, and app stores may use different values when presenting your app."

        |                        | `kaios`            | KaiOS                                |
        |                        | `macos`            | Apple macOS                          |
        |                        | `windows`          | Microsoft Windows                    |
        |                        | `xbox`             | Microsoft Xbox                       |
        | Distribution platforms | `chrome_web_store` | Google Chrome Web Store              |
        |                        | `itunes`           | iTunes App Store                     |
        |                        | `microsoft-inbox`  | Pre-installed with Microsoft Windows |
        |                        | `microsoft-store`  | Microsoft Store                      |
        |                        | `play`             | Google Play Store                    |

## Description

The `screenshots` member is supplementary metadata that does not affect the app's runtime behavior or how browsers present the app.
Screenshots are used in app stores and other distribution platforms to showcase your app's features to potential users.

Distribution platforms may choose how many screenshots to display.

> The images below from the Apple App Store show how screenshots appear in an iphone app listing.

Reviewer: ```suggestion
The images below from the Apple App Store show how screenshots appear in an iPhone app listing.
```

In the gallery view (image on the left), users can scroll through multiple images horizontally.
They can tap any image to view it in full screen (image on the right).

<div style="display: flex; justify-content: center; flex-wrap: wrap;">
  <img src="todoist-iphone-gallery.png" alt="Todoist app in the Apple App Store showing a horizontal gallery of app images that users can scroll through" width="350">
  <img src="todoist-iphone-fullscreen.png" alt="A full-screen view of an image from the horizontal gallery of Todoist app" width="350">
</div>

> The same full-screen image on iPad, as shown below, shows why different screenshots are needed to consider narrow (iPhone) and wide (iPad) form factors of devices:

Reviewer: ", as shown below, " feels a bit unnecessary. 

```suggestion
The same full-screen image on iPad shows why different screenshots are needed for display on narrow (iPhone) and wide (iPad) device form factors:
```

Author: Fair. Updating to:
"The same full-screen image on iPad shows why different screenshots are needed for narrow (iPhone) and wide (iPad) device form factors:"
